### PR TITLE
Fixing initial documentation warnings.

### PR DESCRIPTION
**Pull request by nwagner**
I noticed there were doxygen warnings in the last PR, so I created this one to fix them.  I also reformatted the touched files, so you might want to look at the commits.

### CODE REVIEW

**Review by ChristianReese — APPROVED** [review]

File: docs/pages/mainpage.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
 
 ## Introduction
 
-sACN is a C-language library that implements **ANSI E1.31: Lightweight streaming protocol for
-transport of DMX512 using ACN**, commonly referred to as **Streaming ACN** or **sACN**. The sACN
-library is designed to be portable and scalable to almost any sACN usage scenario, from lightweight
-embedded devices to large-scale data sending operations.
+*Streaming ACN (sACN)* is an ANSI standard for entertainment technology by
```
Comment (samkearney) [inline docs/pages/mainpage.md:5]: This seems like it was fine before? It leads with a description of what's in this repo, not what the standard is.

File: docs/pages/mainpage.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+[ESTA](http://tsp.esta.org) for transmission of DMX512 data over IP networks. sACN is widely used
+in the entertainment industry for real-time control of entertainment technology, especially
+lighting fixtures.
 
-**NOTE**: This open-source implementation is still early in development, and currently only
-receiving sACN is implemented.
+This repository contains a C-language library and a C++ wrapper library for communicating via sACN.
+
+**NOTE**: This open-source implementation is still in development. The merge receiver, source, and
```
Comment (samkearney) [inline docs/pages/mainpage.md:12]: Assuming the implementations will be done before being merged to develop, this isn't necessary.

**Review by ChristianReese — APPROVED** [review]

File: docs/pages/mainpage.md (hunk 1 above)
Comment (nwagner) [reply docs/pages/mainpage.md:5]: Ok.  I can revert that, but add "(with C++ wrappers)".

Comment (samkearney) [reply docs/pages/mainpage.md:5]: Sure... or take the [EtcPal approach](https://etclabs.github.io/EtcPal/docs/v0.3/) and mention the C++ wrappers later. Or just say "a library"

**Pull request merged**